_layouts/default.html

```diff
@@ -2,7 +2,7 @@
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
```

Comment:
This PR is also removing users' ability to zoom _out_ as this should never be necessary, however, they will still be able to zoom _in_ if they choose to. More information about the topic can be found elsewhere.

> minimum-scale | The minimum amount the visitor can zoom on the page. 1.0 does not zoom.
> -- | --
>
> &mdash; https://css-tricks.com/snippets/html/responsive-meta-tag/

As further clarification that this is not a mistake, see the following quote from MDN.

> Usage of `user-scalable=no` can cause accessibility issues to users with visual impairments such as low vision.
>
> &mdash; https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag#Viewport_basics

This means that we're still 🟢 as far as a11y is concerned.